hnsendpoint.go

```diff
@@ -26,6 +26,28 @@ type HNSEndpoint struct {
 	IsRemoteEndpoint   bool              `json:",omitempty"`
 }
 
+// Type of Resource Support in ModifySystem
+type SystemType string
+
+// SystemType const
+const (
+	ContainerType      SystemType = "Container"
+	VirtualMachineType SystemType = "VirtualMachine"
+	HostType           SystemType = "Host"
+)
+
+// EndpointAttachDetachRequest is the structure used to send request to the container to modify the system
+// Supported resource types are Network and Request Types are Add/Remove
+type EndpointAttachDetachRequest struct {
+	ContainerId    string     `json:"ContainerId,omitempty"`

```

Comment:
nit: in Go variables, acronyms should be capitalized.

    ContainerID    string     `json:"ContainerId,omitempty"`